docs/code-search/code-navigation/how-to/adding_lsif_to_workflows.mdx

```diff
@@ -159,8 +131,8 @@ jobs:
           keys:
             - dependencies
       - run: scip-typescript index
-        # this will upload to Sourcegraph.com, you may need to substitute a different command
-        # by default, we ignore failures to avoid disrupting CI pipelines with non-critical errors.
+        //this will upload to Sourcegraph.com, you may need to substitute a different command
```

Comment:
II think the `#` is correct here for comments in YAML syntax

Reply:
This was marked as resolved but I still see `//` instead of `#`.

Reply:
Must have been missed. Fixed it.